src/protocols/rdp/client.h

```diff
@@ -87,6 +87,11 @@
  */
 #define GUAC_RDP_AUDIO_BPS 16
 
+/**
+ * Interval to send the audio input packet to a remote server, in ms.
+ */
+#define GUAC_RDP_AUDIO_INPUT_SEND_INTERVAL 5
```

Comment:
Rather than specify an arbitrary static value, perhaps the interval should be derived from the number of audio frames that the RDP server dictates should be sent in each PDU?